gen_docker_compose_by_variant.py

```diff
@@ -72,16 +71,8 @@ def generate_compose_file(
 
         template_args["ENV"] = os.environ
 
-        for k, v in self.config['quectel_modem'].items():
-            template_args[k] = v
-
-        swarm_key_uri = variant_definitions[variant_type]['SWARM_KEY_URI'][0]
-        parse_result = urlparse(swarm_key_uri)
-        i2c_bus = parse_result.hostname
-        i2c_device = f'/dev/{i2c_bus}'
-
-        template_args['ARCH'] = variant_definitions[variant_type]['CPU_ARCH']
-        template_args['I2C_DEVICE'] = i2c_device
+        template_args["ARCH"] = variant_definitions[variant_type]["CPU_ARCH"]
+        template_args["SWARM_KEY_URI"] = variant_definitions[variant_type]["SWARM_KEY_URI"][0]
```

Comment:
surely changing this will break things? as the i2c bus will not be properly exposed.

also - this needs to be handled at runtime not in the dockerfile env variables or it will break things

Reply:
No, not actually breaks things as we are working in privileged mode for a long time. But Syncrobit Swarm Key is another story, I'll take a look at it in another PR. Otherwise AFAIK there is no variability except Syncrobit devices, right?

Reply:
syncrobit has multiple possible, and helium OG has an onboarding key location that is different from the public key (syncrobit has this too on some units)

but as this filters through to our openfleets where we already have 700+ syncrobit we really need to support it from day 1